fanuc_m10ia_support/launch/test_m10ia7l.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find fanuc_m10ia_support)/launch/load_m10ia7l.launch" />
+  <param name="use_gui" value="true" />
```

Comment:
Could you please make this a private parameter of the `joint_state_publisher` node?